ooni/ui/cli.py

```diff
@@ -224,25 +224,23 @@ def initializeOoniprobe(global_options):
                               should_upload=should_upload,
                               preferred_backend=preferred_backend)
     config.set_initialized()
+    print("ooniprobe is now initialized. You can begin using it!")
 
 def setupGlobalOptions(logging, start_tor, check_incoherences):
     global_options = parseOptions()
 
     config.global_options = global_options
 
-    if not config.is_initialized():
-        log.err("You first need to agree to the informed consent and setup "
-                "ooniprobe to run it.")
```

Comment:
Yeah that is due to how the logging is setup and it "stealing" stdin and stdout.

I think we will have to at some point have a better separation between messages that are "logs" and messages that are meant for the user.